tests/test_install.py

```diff
@@ -41,6 +43,9 @@ def test(tmpdir):
     model_path = os.path.join(tmpdir, 'model.yaml')
     model.to_yaml(model_path)
 
+    # ensure DLL file is unloaded
+    del opensmile.core.SMILEapi.smileapi
```

Comment:
@frankenjoe any other idea what we could do to unload the DLL under Windows (otherwise it cannot be deleted and uninstalling `opensmile` fails, see https://github.com/audeering/audonnx/issues/2#issuecomment-1162749372

Another solution would maybe to simply skip this test under Windows, or to find a feature extractor package that does not require a DLL file.

Reply:
OK, I disabled the test.

Reply:
Interesting tests are now passing and the original issue reported in #33 is not longer a problem.

Reply:
Yes, don't know why ;)